common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -57,7 +57,7 @@
                 MicrosoftStsTokenResponse,
                 TokenResult> {
 
-    private MicrosoftStsOAuth2Configuration mConfig;
+    private MicrosoftStsOAuth2Configuration mConfig;//NOPMD
```

Comment:
Note to myself: this field doesn't need to exist. Should use field in supertype with generic-types accessor.